app/components/pages/SubmissionWizard/steps/Disclosure/index.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Box } from 'grid-styled'
+import { Checkbox } from '@pubsweet/ui'
+import { th } from '@pubsweet/ui-toolkit'
+
+import { FormH3 } from '../../../../ui/atoms/FormHeadings'
+import ValidatedField from '../../../../ui/atoms/ValidatedField'
+
+const StyledLink = styled.a`
+  color: ${th('colorPrimary')};
+  text-decoration: none;
+`
+
+const RegularP = styled.p`
+  font-size: ${th('fontSizeBase')};
+  line-height: ${th('fontLineHeightBase')};
+`
+
+const SmallP = styled.p`
+  font-size: ${th('fontSizeBaseSmall')};
+  line-height: ${th('fontLineHeightBaseSmall')};
+  color: ${th('colorTextSecondary')};
```

Comment:
These look like good candidates to be atoms. Let's make sure that we use the atoms anywhere we would be using `RegularP`, `SmallP` and `StyledLink`

Reply:
Yeah completely agreed. Especially since we have these elsewhere in the codebase as well (within [PersonPod](https://github.com/elifesciences/elife-xpub/blob/66c00a84bf03c611f873c0cd1e239200aa5cb48f/app/components/ui/atoms/PersonPod.js#L75))

I've raised an issue in pubsweet, as the paragraph components should probably be added in there - https://gitlab.coko.foundation/pubsweet/pubsweet/issues/423

I'm unsure about `StyledLink`, but I should probably be using pubsweet's [`Link` component](https://gitlab.coko.foundation/pubsweet/pubsweet/blob/master/packages/ui/src/atoms/Link.js) & overriding styling to remove text-decoration.

@hdrury1 Am I right in saying that this is going to be a common pattern across the `elife-xpub` app? That all links within text will be styled like this?
```
color: ${th('colorPrimary')};
text-decoration: none;
```

Reply:
Correct @jsms90